src/content/docs/it/install/manual.mdx

```diff
@@ -0,0 +1,197 @@
+- **Terminale** - Astro viene utilizzato tramite la sua interfaccia da riga di comando (CLI).
+
+#### Installazione
+
+Se preferisi non usare il nostro strumento CLI automatico `create astro`, puoi impostare tu stesso il tuo progetto seguendo questa guida.
+
+## 1. Crea la tua cartella
+
+Crea una cartella vuota con il nome del tuo progetto, e naviga al suo interno.
+
+```bash
+mkdir il-mio-progetto-astro
+cd il-mio-progetto-astro
+```
+
+Una volta all'interno della cartella, crea il file `package.json` del tuo progetto. Questo servirà a gestire le dipendenze del tuo progetto, incluso Astro. Se questo formato di file non ti è familiare, esegui il seguente comando per crearne uno.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm init --yes
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm init 
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn init --yes
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+
+## 2. Installa Astro
+
+Per prima cosa, installa le dipendenze di un progetto Astro all'interno del tuo progetto.
+
+:::note[Importante]
+Astro deve essere installato localmente, non globalmente. Assicurati di *non* star eseguendo `npm install -g astro`, `pnpm add -g astro` oppure `yarn add global astro`.
+:::
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install astro
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm install astro 
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Poi, sostituisci qualunque sezione segnaposto "scripts" all'interno del tuo file `package.json` con il seguente codice:
+
+```json title="package.json" del={2} ins={3-6}
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview"
+  },
+```
+
+Userai questi script più avanti nella guida per far partire Astro ed eseguire i suoi vari comandi.
+
+## 3. Crea la tua prima pagina
+
+Nel tuo editor di testo, crea un nuovo file nella tua cartella al percorso `src/pages/index.astro`. Questa sarà la tua prima pagina Astro nel progetto.
+
+Per questa guida, copia e incolla il seguente blocco di codice (inclusi i tre trattini `---`) all'interno del tuo nuovo file:
+
+```astro title="src/pages/index.astro"
+---
+// Benvenuto/a ad Astro! Tutto ciò che è delimitato da questi tre trattini
+// è il tuo "avantesto del componente". Non viene mai eseguito nel browser.
+console.log('Questo viene eseguito nel tuo terminale, non nel browser!');
+---
+<!-- Qua sotto c'è il tuo "template del componente". È solo HTML, con
+     un po' di magia per aiutarti a costruire ottimi template. -->
+<html>
+  <body>
+    <h1>Hello, World!</h1>
+  </body>
+</html>
+<style>
+  h1 {
+    color: orange;
+  }
+</style>
+```
+
+## 4. Crea il tuo primo asset statico
```

Comment:
Do you think we can translate "asset" with "risorsa"? Just asking

Reply:
Yeah it's possible 👍🏻
I left it there cause I've heard it being used in a lot of places. I can change it if you think it's better